maxim/logger/pydantic_ai/client.py

```
@@ -0,0 +1,735 @@
+                    "name": tool_call.get('function', {}).get('name', 'unknown'),
+                    "args": tool_call.get('function', {}).get('arguments', {}),
+                    "tool_call_id": tool_call.get('id', str(uuid.uuid4()))
+                })
+                
+    except Exception as e:
+        scribe().debug(f"[MaximSDK] Error extracting tool calls: {e}")
+    
+    return tool_calls
+
+
+def start_session_trace(maxim_logger: Logger, name: str = "Pydantic AI Session"):
```

Ruff (0.12.2)

Missing return type annotation for public function `start_session_trace`

(ANN201)